src/rgw/rgw_rest_log.cc

```diff
@@ -815,6 +815,130 @@ void RGWOp_MDLog_Status::send_response()
   flusher.flush();
 }
 
+class RGWOp_MDLog_CompareStatus : public RGWRESTOp {
+  rgw_meta_sync_status status;
+  RGWMetadataLogInfo info;
+  list<cls_log_entry> entries;
+  string last_marker;
+  bool truncated;
+  ceph::real_time *current_timestamp, *latest_timestamp;
```

Comment:
pointers to time variables?  that looks suspicious

Reply:
Thinking of changing it to vector of real_time to populate the values